Hmac.js

```diff
@@ -48,12 +54,42 @@ export class Hmac {
    *
    * @param {object} options - The options to use.
    * @param {Uint8Array} options.data - The data to sign as a Uint8Array.
+   * @param {boolean} [options.useCache=true] - Enable the use of a cache.
    *
    * @returns {Promise<string>} The base64url-encoded signature.
    */
-  async sign({data}) {
+  async sign({data, useCache = true}) {
```

Comment:
Not sure how much more you want to work on this, but you could factor out the common DRY caching part.  sign and verify could call a common function with a key prefix string and a function that does the work.  Everything else is the same.  Or since this code is short, just not worry about it for now.